content/developer/reference/frontend/odoo_editor.rst

```diff
@@ -0,0 +1,223 @@
+===========
+Odoo Editor
```

Comment:
No its name is just Odoo Editor. A priori if someone has to work within the editor, they're more than likely to know it's called Odoo Editor (since it's relatively low level).